packages/sdk-node/src/authenticateRequest.ts

```diff
@@ -1,8 +1,13 @@
 import { Clerk, constants, RequestState } from '@clerk/backend';
+import cookie from 'cookie';
 import { Request as ExpressRequest } from 'express';
 
 import { ClerkMiddlewareOptions } from './types';
 
+const parseCookies = (req: ExpressRequest) => {
+  return cookie.parse(req.headers['cookie'] || '');
```

Comment:
❓ shouldn't we return `{}` as default value since we are accessing the return value as an object?